src/app/dashboard/whitelist/actions.js

```diff
@@ -0,0 +1,273 @@
+//@flow
+
+import * as ui from 'polymath-ui'
+import { TransferManager, SecurityToken } from 'polymathjs'
+import type { Investor } from 'polymathjs/types'
+import { formName as userFormName } from './components/userForm'
+import { formName as editInvestorsFormName } from './components/editInvestorsForm'
+import type { GetState } from '../../../redux/reducer'
+import type { ExtractReturn } from '../../../redux/helpers'
+
+//ac_ = actionCreator
+export const TRANSFER_MANAGER = 'dashboard/whitelist/TRANSFER_MANAGER'
+export const ac_transferManager = (transferManager: TransferManager) => ({ type: TRANSFER_MANAGER, transferManager: transferManager })
```

Comment:
Use shorter `transferManager`, not `transferManager: transferManager`

Reply:
fixed